packages/block-editor/src/components/link-control/style.scss

```diff
@@ -51,13 +51,6 @@ $preview-image-height: 140px;
 	position: relative;
 }
 
-// If the input doesn't have a visible label then
-// we need to expand the input itself to occupy
-// the full available horizontal space.
-.block-editor-link-control__search-input.has-no-label .block-editor-url-input__input {
-	flex: 1;
-}
```

Comment:
Minor refactoring: This style is unnecessary because the `has-no-label` class does not exist under any conditions.